about/timescaledb-editions.md

```diff
@@ -158,63 +158,9 @@ You can access a hosted version of TimescaleDB Community Edition through
     <td>✅</td>
   </tr>
   <tr>
-    <td><strong>Distributed hypertables</strong></td>
-  </tr>
-  <tr>
-    <td><a href="https://docs.timescale.com/api/latest/distributed-hypertables/create_distributed_hypertable/">create_distributed_hypertable</a></td>
-    <td>❌</td>
-    <td>✅</td>
-  </tr>
-  <tr>
-    <td><a href="https://docs.timescale.com/api/latest/distributed-hypertables/add_data_node/">add_data_node</a></td>
-    <td>❌</td>
-    <td>✅</td>
-  </tr>
-  <tr>
-    <td><a href="https://docs.timescale.com/api/latest/distributed-hypertables/alter_data_node/">alter_data_node</a></td>
-    <td>❌</td>
-    <td>✅</td>
-  </tr>
-  <tr>
-    <td><a href="https://docs.timescale.com/api/latest/distributed-hypertables/attach_data_node/">attach_data_node</a></td>
-    <td>❌</td>
-    <td>✅</td>
-  </tr>
-  <tr>
-    <td><a href="https://docs.timescale.com/api/latest/distributed-hypertables/detach_data_node/">detach_data_node</a></td>
-    <td>❌</td>
-    <td>✅</td>
-  </tr>
-  <tr>
-    <td><a href="https://docs.timescale.com/api/latest/distributed-hypertables/delete_data_node/">delete_data_node</a></td>
-    <td>❌</td>
-    <td>✅</td>
-  </tr>
-  <tr>
-    <td><a href="https://docs.timescale.com/api/latest/distributed-hypertables/distributed_exec/">distributed_exec</a></td>
-    <td>❌</td>
-    <td>✅</td>
-  </tr>
-  <tr>
-    <td><a href="https://docs.timescale.com/api/latest/distributed-hypertables/set_number_partitions/">set_number_partitions</a></td>
-    <td>❌</td>
-    <td>✅</td>
-  </tr>
-  <tr>
-    <td><a href="https://docs.timescale.com/api/latest/distributed-hypertables/set_replication_factor/">set_replication_factor</a></td>
-    <td>❌</td>
-    <td>✅</td>
-  </tr>
-  <tr>
-    <td><a href="https://docs.timescale.com/api/latest/distributed-hypertables/copy_chunk_experimental/">copy_chunk</a></td>
-    <td>❌</td>
-    <td>✅</td>
-  </tr>
-  <tr>
-    <td><a href="https://docs.timescale.com/api/latest/distributed-hypertables/move_chunk_experimental/">move_chunk</a></td>
-    <td>❌</td>
-    <td>✅</td>
+    <td colspan="3"><strong>Distributed hypertables</strong>: This feature is deprecated in all editions.</td>
```

Comment:
Hi @ramonguiu, are you ok with me removing the list for a deprecated features?

Reply:
Thank you @billy-the-fish - makes sense to me, no reason to keep those deprecated features around any more. Not sure if we should even keep this line around anymore, but your call. 

If we keep the line, should we add something like the following? (non-blocking comment for your consideration) 

```diff
+     <td colspan="3"><strong>Distributed hypertables</strong>: This feature is deprecated in all editions after TimescaleDB 2.13. Learn more about it [here](https://github.com/timescale/timescaledb/blob/2.14.0/docs/MultiNodeDeprecation.md).</td>
```